pkg/server/controllers.go

```diff
@@ -227,34 +227,34 @@ func readCA(file string) ([]byte, error) {
 	return rootCA, err
 }
 
-func (s *Server) installNamespaceScheduler(ctx context.Context, workspaceLister tenancylisters.ClusterWorkspaceLister, clientConfig clientcmdapi.Config, server *genericapiserver.GenericAPIServer) error {
-	kubeClient, err := kubernetes.NewClusterForConfig(server.LoopbackClientConfig)
+func (s *Server) installNamespaceScheduler(ctx context.Context, config *rest.Config) error {
+	kubeClient, err := kubernetes.NewClusterForConfig(config)
```

Comment:
why are we creating multiple clients instead of sharing the same client?

Reply:
this will end having multiple TCP connections, can we make the client part of the struct? 
not asking for doing it in this PR,